Chaos/ChampionDrakathMerge.cs

```diff
@@ -0,0 +1,147 @@
+/*
+name: Champion Drakath Merge
+description: This bot will farm the items belonging to the selected mode for the Champion Drakath Merge [2055] in /championdrakath
+tags: champion, drakath, merge, championdrakath, empowered, original, chaos, avengers, greatsword, avenger, polished, dragon, control, supreme, arcane, colorful, rose, royal
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Story/LordsofChaos/Core13LoC.cs
+//cs_include Scripts/Chaos/DrakathsArmor.cs
+//cs_include Scripts/Chaos/ChaosAvengerPreReqs.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/Nation/CoreNation.cs
+//cs_include Scripts/Good/BLoD/CoreBLOD.cs
+//cs_include Scripts/Story/BattleUnder.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class ChampionDrakathMerge
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv = new();
+    private static CoreAdvanced sAdv = new();
+    public Core13LoC LOC => new();
+    private DrakathArmorBot DAB = new();
+    private ChaosAvengerClass CAV = new();
+
+    public List<IOption> Generic = sAdv.MergeOptions;
+    public string[] MultiOptions = { "Generic", "Select" };
+    public string OptionsStorage = sAdv.OptionsStorage;
+    // [Can Change] This should only be changed by the author.
+    // If true, it will not stop the script if the default case triggers and the user chose to only get mats
+    private bool dontStopMissingIng = false;
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.BankingBlackList.AddRange(new[] { "Drakath Armor", "Champion Drakath Insignia", "Original Drakath Armor", "Blade of Chaos", "Chaos Avenger's Greatsword", "Chaos Avenger Armor", "Legendary Sword of Dragon Control", "The Supreme Arcane Staff", "Discordia Rose of Chaos", "Chaos Rose" });
+        Core.SetOptions();
+
+        BuyAllMerge();
+        Core.SetOptions(false);
+    }
+
+    public void BuyAllMerge(string? buyOnlyThis = null, mergeOptionsEnum? buyMode = null)
+    {
+        LOC.Hero();
+        if (Core.isCompletedBefore(8301))
```

Comment:
!Core as we're checking if its *not* complete and returning if its not complete.

!Core = not
Core = is